internal/cluster/distribution/eks/eksprovider/workflow/amazon.go

```diff
@@ -156,6 +156,7 @@ func packageCFError(err error, stackName string, clientRequestToken string, clou
 }
 
 // EKSActivityInput holds common input data for all activities
+// Deprecated! Use the AWSCommonActivityInput from "github.com/banzaicloud/pipeline/internal/cluster/infrastructure/aws/awsworkflow" instead
```

Comment:
**Question**: are we planning to update the other references in a separate PR or what is our strategy here?

Reply:
Once the Cadence SDK is ready , this will probably disappear entirely.